cmake/config/Config.cmake.in

```diff
@@ -146,7 +146,10 @@ SET(DEAL_II_USER_DEFINITIONS_RELEASE "@DEAL_II_USER_DEFINITIONS_RELEASE@")
 
 SET(DEAL_II_MPIEXEC "@MPIEXEC@")
 SET(DEAL_II_MPIEXEC_NUMPROC_FLAG "@MPIEXEC_NUMPROC_FLAG@")
-SET(DEAL_II_MPIEXEC_PREFLAGS "@MPIEXEC_PREFLAGS@")
+# we must avoid possible spaces in MPIEXEC_PREFLAGS as those will be
+# set into quotation marks when generating the build.make instructions
+# -> replace them by semicolons here
+SEPARATE_ARGUMENTS(DEAL_II_MPIEXEC_PREFLAGS NATIVE_COMMAND "@MPIEXEC_PREFLAGS@")
```

Comment:
This is the wrong place to fix it (we need the `SEPARATE_ARGUMENTS` already during the deal.II cmake configure). I'll push a suggestion to your branch momentarily. :-)